INDLinkLabel.swift

```diff
@@ -31,6 +31,14 @@ import UIKit
     
     /// Called when a link is long pressed.
     optional func linkLabel(label: INDLinkLabel, didLongPressLinkWithURL URL: NSURL)
+
+    /**
```

Comment:
fixed in https://github.com/cdzombak/INDLinkLabel/commit/6a3163f3ea9a134c5b6e63dbf9d000a3c5794f99